astro_2.0/features/pages/nestedDaoPagesContent/DelegatePageContent/components/DelegateGroupTable/DelegateGroupTable.module.scss

```diff
@@ -0,0 +1,87 @@
+@import 'styles/typography';
+@import 'styles/mixins';
+
+.root {
+  display: flex;
+  flex-direction: column;
+}
+
+.header,
+.row,
+.actionsRow {
+  display: grid;
+  grid-column-gap: 12px;
+  grid-template-columns: repeat(3, 1fr) 0.7fr;
+
+  padding: 12px 24px;
+
+  > div {
```

Comment:
We should avoid tags in css when possible.

Reply:
fixed